src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,47 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  width: getWidth($cell-size, $gap, $padding);
+  margin: 0 auto;
```

Comment:
BEM doesn't allow us to do so. Better stylize the parent element to center the calendar.